strings/tips.txt

```diff
@@ -210,7 +208,7 @@ As the Blob, removing strong blobs, resource nodes, factories, and nodes will gi
 As the Blob, talking will send a message to all other overminds and all Blobbernauts, allowing you to direct attacks and coordinate.
 As a Blobbernaut, you can communicate with overminds and other Blobbernauts via :b.
 As a Blobbernaut, your HUD shows your health and the core health of the overmind that created you.
-As a Revolutionary, you cannot convert a head of staff or someone who has a mindshield implant, such as a security officer or those they implant. Implants can however be surgically removed, and do not carry over with cloning. Take control of medbay to keep control of conversions!
+As a Revolutionary, you cannot convert a head of staff or someone who has a mindshield implant, such as a security officer or those they implant. Implants can however be surgically removed. Take control of cargo to keep control of conversions!
```

Comment:
Wasnt this saying to keep control of medbay because it also says that implants can be surgically removed?

Reply:
More so that when a seccie got cloned you could convert them before they ever got control of their mob.

Reply:
If you're gonna refocus the tip on cargo you may want to mention the implant crates